kairon/cli/mail_channel_read.py

```
@@ -0,0 +1,29 @@
+import json
```

Ruff (0.8.0)

`json` imported but unused

Remove unused import: `json`

(F401)